cluster-stamp.json

```diff
@@ -1813,6 +1820,36 @@
                 "windowSize": "PT1M"
             }
         },
+        {
+            "condition": "[variables('isUsingAzureRBACasKubernetesRBAC')]",
+            "type": "Microsoft.Authorization/roleAssignments",
+            "apiVersion": "2018-09-01-preview",
+            "name": "[guid(concat('aad-admin-group', resourceId('Microsoft.ContainerService/managedClusters', variables('clusterName'))))]",
+            "dependsOn": [
+                "[resourceId('Microsoft.ContainerService/managedClusters', variables('clusterName'))]"
+            ],
+            "scope": "[concat('Microsoft.ContainerService/managedClusters/', variables('clusterName'))]",
+            "properties": {
+                "roleDefinitionId": "[concat('/subscriptions/', subscription().subscriptionId, '/providers/Microsoft.Authorization/roleDefinitions/', variables('clusterAdminRoleId'))]",
+                "principalId": "[parameters('clusterAdminAadGroupObjectId')]",
+                "principalType": "Group"
+            }
+        },
+        {
+            "condition": "[variables('isUsingAzureRBACasKubernetesRBAC')]",
+            "type": "Microsoft.Authorization/roleAssignments",
+            "apiVersion": "2018-09-01-preview",
+            "name": "[guid(concat('aad-a0005-reader-group', resourceId('Microsoft.ContainerService/managedClusters', variables('clusterName'))))]",
+            "dependsOn": [
+                "[resourceId('Microsoft.ContainerService/managedClusters', variables('clusterName'))]"
+            ],
+            "scope": "[concat('/subscriptions/', subscription().subscriptionId, '/resourcegroups/', resourceGroup().name, '/providers/Microsoft.ContainerService/managedClusters/', variables('clusterName'), '/namespaces/a0005')]",
+            "properties": {
+                "roleDefinitionId": "[concat('/subscriptions/', subscription().subscriptionId, '/providers/Microsoft.Authorization/roleDefinitions/', variables('clusterReaderRoleId'))]",
+                "principalId": "[parameters('a0005NamespaceReaderAadGroupObjectId')]",
+                "principalType": "Group"
```

Comment:
Can you also grant this group the **Azure Kubernetes Service Cluster User Role** on the cluster right away as well?  That way that completes the story without the user needing to do even more work outside of this process.

Reply:
Done !